pkg/types/v2/metadata.go

```diff
@@ -91,7 +91,13 @@ func (mt MetadataTest) Run(driver drivers.Driver) *types.TestResult {
 
 	for _, pair := range mt.Env {
 		if val, ok := imageConfig.Env[pair.Key]; ok {
-			if pair.Value != val {
+			match := false
```

Comment:
nit: declaring this with `var match bool` is more idiomatic. Go will default it to `false`

Reply:
@nkubala Updated. Thanks!